cmds/fspinfo/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/linuxboot/fiano/pkg/fsp"
+	"github.com/linuxboot/fiano/pkg/uefi"
+)
+
+func main() {
+	flag.Parse()
+
+	if flag.Arg(0) == "" {
+		panic("missing filename")
```

Comment:
maybe I should not use `panic` here

Reply:
log.Fatalf

Reply:
I already replaced it with fmt.Fprintf and os.Exit... I am normally wary of log.Fatal because it doesn't honor deferred functions, but in this case it's not an issue. I'll change it to log.Fatal

Reply:
done